src/opm/output/eclipse/Summary.cpp

```diff
@@ -891,22 +878,24 @@ static const std::unordered_map< std::string, ofun > funs = {
     { "WOIR", rate< rt::oil, injector > },
     { "WGIR", rate< rt::gas, injector > },
     { "WNIR", rate< rt::solvent, injector > },
-    { "WCIR", rate< rt::wat, injector, polymer > },
+    { "WCIR", rate< rt::polymer, injector > },
     { "WVIR", sum( sum( rate< rt::reservoir_water, injector >, rate< rt::reservoir_oil, injector > ),
                        rate< rt::reservoir_gas, injector > ) },
 
     { "WWIT", mul( rate< rt::wat, injector >, duration ) },
     { "WOIT", mul( rate< rt::oil, injector >, duration ) },
     { "WGIT", mul( rate< rt::gas, injector >, duration ) },
     { "WNIT", mul( rate< rt::solvent, injector >, duration ) },
-    { "WCIT", mul( rate< rt::wat, injector, polymer >, duration ) },
+    { "WCIT", mul( rate< rt::polymer, injector >, duration ) },
     { "WVIT", mul( sum( sum( rate< rt::reservoir_water, injector >, rate< rt::reservoir_oil, injector > ),
                         rate< rt::reservoir_gas, injector > ), duration ) },
 
     { "WWPR", rate< rt::wat, producer > },
     { "WOPR", rate< rt::oil, producer > },
     { "WGPR", rate< rt::gas, producer > },
     { "WNPR", rate< rt::solvent, producer > },
+    { "WCPR", rate< rt::polymer, producer > },
+    { "WCPC", div( rate< rt::polymer, producer >, rate< rt::wat, producer >) },
```

Comment:
This `*CPCP` quantity is going to request that we generate a unit of
```
  mass rate
-------------
 volume rate
```
That unit isn't currently encoded in [`div_unit`](https://github.com/OPM/opm-common/blob/a067352ce0c1eab4d9f3f315ab60b5a9395df9a8/src/opm/output/eclipse/Summary.cpp#L274), so we'll end up getting [`measure::identity`](https://github.com/OPM/opm-common/blob/a067352ce0c1eab4d9f3f315ab60b5a9395df9a8/src/opm/output/eclipse/Summary.cpp#L299)&mdash;effectively no unit of measure.  In reality I think we should have a density-like `mass / volume` unit instead.

Unfortunately, no such unit is currently defined in the [`UnitSystem::measure`](https://github.com/OPM/opm-common/blob/a067352ce0c1eab4d9f3f315ab60b5a9395df9a8/opm/parser/eclipse/Units/UnitSystem.hpp#L42) table so fully fixing this also requires defining that unit.

Reply:
Shouldn't be too much work to fix. I can look at it on Monday.

Reply:
@bska I have added what I thought you suggested. I kept the name polymer_density. That is maybe a bit specific since mass rate / volume rate of water could be used for other purposes.